src/operation.rs

```diff
@@ -111,13 +111,37 @@ pub fn pauli_z(target: usize) -> Operation {
     }
 }
 
+pub fn toffoli(controls: &Vec<usize>, target: usize) -> Operation {
```

Comment:
controls could be written as a slice instead but this works so no need t ochange